jsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "baseUrl": "./src",
+    "experimentalDecorators": true
```

Comment:
this is a duplicate of the setting already in `.vscode/settings.json`, i chose the file inside a folder that's explicitly named vscode for clarity. jsconfig.json is a file i really don't like if i'm honest!

Reply:
VSCode doesn't suggest correctly files in your project without that settings )

Maybe just experimentalDecorators is option that can be also configured though `settings.json`

Reply:
Yes I was only talking about experimentalDecorators.

I have some local settings on a branch I'm working on for docker images and cross browser testing.

E.g.g

```json
{
  "compilerOptions": {
    "plugins": [
      {
        "name": "typescript-snapshots-plugin",
        "snapshotDir": "./",
        "snapshotCallIdentifiers": [
          "toMatchSnapshot",
          "toThrowErrorMatchingSnapshot",
          "matchSnapshot",
        ]
      }
    ],
  }
}
```

I'm happy to add the `baseUrl` here too, sounds useful!